mkosi/__init__.py

```diff
@@ -286,6 +286,21 @@ def configure_autologin(state: MkosiState) -> None:
                 """  # noqa: E501
             )
 
+        if state.config.architecture == Architecture.arm64:
+            dropin = state.root / "usr/lib/systemd/system/serial-getty@ttyAMA0.service.d/autologin.conf"
+            with umask(~0o755):
+                dropin.parent.mkdir(parents=True, exist_ok=True)
+            with umask(~0o644):
+                dropin.write_text(
```

Comment:
I think this also needs a `dedent`. Oh, so does the code above.

Those ~~two~~ three copies need to be unified.

Reply:
> I think this also needs a dedent. Oh, so does the code above.

I can add that. I think I mentioned it, when the code got refactored the last time, but it was declined because the unit file doesn't care about initial whitespace

> Those ~~two~~ three copies need to be unified.

I think it's only two, because the topmost has three `--keep-baud` settings while the bottom two have four. I have really no clue how finicky real-world ttys are in this regard and only copied the bottom one. For this some feedback from  @jgouly would be needed, since I have no hardware to test this presently.